retry/lib/retry.dart

```diff
@@ -177,8 +177,8 @@ Future<T> retry<T>(
   double randomizationFactor = 0.25,
   Duration maxDelay = const Duration(seconds: 30),
   int maxAttempts = 8,
-  FutureOr<bool> Function(Exception)? retryIf,
-  FutureOr<void> Function(Exception)? onRetry,
+  FutureOr<bool> Function(Exception, StackTrace)? retryIf,
+  FutureOr<void> Function(Exception, StackTrace)? onRetry,
```

Comment:
This could be useful, but is it worth doing a breaking change over this?

Maybe we can make a new optional parameter instead of `onRetry` or to complement `onRetry`. Like `logRetry`?

Reply:
Updated.